src/web/gwc/src/main/java/org/geoserver/gwc/web/layer/CachedLayersPage.java

```diff
@@ -323,9 +329,77 @@ protected Component headerPanel() {
+
+                    @Override
+                    public void onClick(AjaxRequestTarget target) {
+                        dialog.setTitle(
+                                new ParamResourceModel(
+                                        "confirmGwcTruncateTitle", CachedLayersPage.this));
+                        dialog.setDefaultModel(getDefaultModel());
+
+                        dialog.showOkCancel(
+                                target,
+                                new GeoServerDialog.DialogDelegate() {
+                                    private static final long serialVersionUID = 1L;
+                                    private TruncateAllRequest truncateAllRequest;
+
+                                    @Override
+                                    protected Component getContents(final String id) {
+                                        Label confirmLabel =
+                                                new Label(
+                                                        id,
+                                                        "You are about to wipe GWC cache clean, Do you want to continue?");
+                                        confirmLabel.setEscapeModelStrings(
+                                                false); // allow some html inside, like
+                                        // <b></b>, etc
+                                        return confirmLabel;
+                                    }
+
+                                    @Override
+                                    protected boolean onSubmit(
+                                            final AjaxRequestTarget target,
+                                            final Component contents) {
+
+                                        GWC facade = GWC.get();
+                                        try {
+                                            truncateAllRequest = facade.truncateAll();
+                                        } catch (Exception e) {
+                                            error(e.getMessage());
+                                            log.error("An Error while clearing GWC cache", e);
+                                            return false;
+                                        }
+                                        return true;
+                                    }
+
+                                    @Override
+                                    public void onClose(final AjaxRequestTarget target) {
+                                        target.add(table);
+                                        if (truncateAllRequest != null)
+                                            if (truncateAllRequest.getTrucatedLayers().length()
+                                                    == 0) warn("GWC Cache already empty");
+                                            else
+                                                info(
+                                                        "Cleared GWC cache for "
```

Comment:
Internationalize, see ParamResourceModel and the property files GeoServerApplication.properties.

Reply:
will do